docs/user-guide/building-assistants.rst

```diff
@@ -8,8 +8,7 @@ Tutorial: Building Assistants
 .. edit-link::
 
 After following the basics of setting up an assistant in the `Rasa Tutorial <https://rasa.com/docs/rasa/user-guide/rasa-tutorial/>`_, we'll
-now walk through building a basic FAQ chatbot and then build a bot that can handle
-contextual conversations.
+now walk through building a basic FAQ assistant.
```

Comment:
should this not say the original thing again? Since it's still in the same page. Let's discussing moving them to separate pages after this is merged

Reply:
sorry! thought I reverted that.